malcontent.yaml

```diff
@@ -38,11 +38,11 @@ test:
         - openssl
         - crane
   pipeline:
-    - name: Verify malcontent on itself
+    - name: Verify malcontent version
       runs: |
         set -o pipefail
-        output=$(mal analyze /usr/bin/mal)
```

Comment:
This output should be empty so I'll need to take a closer look at the regression. Either way, this takes too long and we removed this as a test in the `malcontent` repository.